config_example.sh

```diff
@@ -23,6 +23,11 @@ set -x
 # NOTE: If a checkout already exists in $HOME, it won't be re-created.
 # NOTE: You must set CUSTOM_REPO_FILE to build some OpenShift images, e.g. ironic ones.
 
+# Use <IMAGE_NAME>_EXTRA_PACKAGES to set the path (relative to dev-scripts or
+# absolute) to a file with extra packages to install in the image, one per line.
+# For example:
+# export IRONIC_IPA_DOWNLOADER_EXTRA_PACKAGES=ipa-downloader-extra-pkgs.txt
```

Comment:
I'm not sure this is a good example - AFAICS the IPA downloader Dockerfile doesn't support EXTRA_PACKAGES?

https://github.com/openshift/ironic-ipa-downloader/blob/master/Dockerfile.ocp#L6

vs

https://github.com/openshift/ironic-image/blob/master/Dockerfile.ocp#L22

So unless this is something now supported in the base image, we should probably list the containers which will work with this interface in the example config?

Reply:
you're right, this is a leftover from another era, thanks, I'll change it